### DISCUSSION TITLE
Pronounciation assessment

### DISCUSSION THREAD

**Discussion by sates52** [openai/whisper#747] [Q&A]
Hi,

Is there any way to get AccuracyScore, FluencyScore ve CompletenessScore from whisper API?

I want to build something like  Microsoft's pronunciation assessment tool.

Thanks

**Comment by amandeepbaberwal** [comment]
@sates52 did you find what you were looking for?

**Comment by Uchihinha** [comment]
I'm interested in it too

**Comment by atdevten** [comment]
Any solution sir?

**Comment by Tsh-kazi** [comment]
Hello! Good evening
Sorry for the late reply, but I really don't think I will be able to help
you because I stopped working on the project and I was not really much
successful.
